package/yast2-ruby-bindings.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Tue Oct 23 09:52:53 UTC 2018 - jreidinger@suse.com
+
+- Fix crash caused by reading external files in ASCII encoding
```

Comment:
That's the immediate cause of the crash. But why is that?
Is glibc-locale.rpm missing? (no)
What is the ENV locale setting?
Why did that happen when the user installed in German?
Anyway, we should mention the more important thing:
> Fix encoding-related problems by assuming that file contents is always UTF-8.